.gitignore

```diff
@@ -15,3 +15,4 @@ debug.*
 *~
 *.bak
 make.in
+quick.rc
```

Comment:
I think we need quick.rc, the one in `quick-cmake/`. Shouldn't ignore it.

Reply:
Fixed.